src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts

```diff
@@ -423,8 +426,7 @@ export class UbsAdminTariffsCardPopUpComponent implements OnInit, OnDestroy {
               stationNames: this.selectedStation.map((it) => it.name),
               regionName: this.region.value,
               regionEnglishName: this.regionEnglishName,
-              locationNames: this.selectedCities.map((it) => it.location),
-              locationEnglishNames: this.selectedCities.map((it) => it.englishLocation),
+              locationNames: this.selectedCities.map((it) => (this.currentLanguage === Language.UA ? it.location : it.englishLocation)),
```

Comment:
The same
Replace `this.currentLanguage === Language.UA ? item.nameUk : item.nameEn` with function 'getLangValue()' from Language service